MonoGame.Framework/Platform/Graphics/TextureCollection.DirectX.cs

```diff
@@ -26,28 +26,23 @@ private void ClearTargets(RenderTargetBinding[] targets, SharpDX.Direct3D11.Comm
             // NOTE: We make the assumption here that the caller has
             // locked the d3dContext for us to use.
 
-            // We assume 4 targets to avoid a loop within a loop below.
```

Comment:
Why were we avoiding a "loop within a loop" in the first place?

Reply:
Goes all the way back to here https://github.com/MonoGame/MonoGame/commit/271f68a4b6d2213e69633156bd19df28acd74564#diff-5a4587d4df5e2194d9d2cf907ce2042388623f47c356004626f72e8255ab96bf